intake_erddap/erddap.py

```diff
@@ -198,7 +216,55 @@ def _close(self):
 
     def _load(self):
         e = self.get_client()
-        self._dataframe: pd.DataFrame = e.to_pandas()
+        self._dataframe: pd.DataFrame = e.to_pandas(
+            requests_kwargs={"timeout": 60}, **self.open_kwargs
+        )
+        if self._mask_failed_qartod:
+            self.run_mask_failed_qartod()
+        if self._dropna:
+            self.run_dropna()
+
+    @property
+    def data_cols(self):
+        """Columns that are not axes, coordinates, nor qc_agg columns."""
+
+        import cf_pandas  # noqa: F401
```

Comment:
The reason was because I thought cf-pandas should only be optionally required, but then I realized that it was already required so I just moved it to the top.